### PR TITLE
Add DisplayBox component

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

This PR includes the following proposed change(s):

- Add display box component for highlighting or separating information

DisplayBox

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested locally with storybook

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/shared-components/src/components/display-box/DisplayBox.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./DisplayBox.css";
+
+export default function DisplayBox({ styling, icon, element }) {
+  return (
+    <div className={`display-box ${styling}`}>
+      <div className={"display-icon"}>{icon}</div>
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/shared-components/src/components/display-box/DisplayBox.js:9]: dont need `{}`
```suggestion
      <div className="display-icon">{icon}</div>
```

File: src/frontend/shared-components/src/components/display-box/DisplayBox.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <div className={"display-element"}>{element}</div>
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/shared-components/src/components/display-box/DisplayBox.js:10]: same here
```suggestion
      <div className="display-element">{element}</div>
```

File: src/frontend/shared-components/src/components/display-box/DisplayBox.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { MdPerson } from "react-icons/md";
+
+import testBasicSnapshot from "../../TestHelper";
+import DisplayBox from "./DisplayBox";
+import { getTestTable } from "../../modules/displayBoxTestData";
+
+const tableData = getTestTable();
+const icon = <MdPerson size={32} />;
+
+describe("Table Component", () => {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/shared-components/src/components/display-box/DisplayBox.test.js:11]: isn't this display box?? should be:
```
describe("DisplayBox Component", () => {});
```

File: src/frontend/shared-components/src/components/display-box/DisplayBox.test.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  test("Matches the without icon snapshot", () => {
```
Comment (akroon3r) [inline src/frontend/shared-components/src/components/display-box/DisplayBox.test.js:12]: Matches the 'without icon' snapshot?

Comment (sdevalapurkar-bcgov) [reply src/frontend/shared-components/src/components/display-box/DisplayBox.test.js:12]: or 
```
Matches the snapshot without icon
```

File: src/frontend/shared-components/src/components/display-box/DisplayBox.test.js (hunk 3 above)
Comment (adodge-bcg) [reply src/frontend/shared-components/src/components/display-box/DisplayBox.test.js:11]: Copy pasta, fix inc

**Pull request merged**